rllib/utils/metrics/tests/test_metrics_logger.py

```diff
@@ -275,25 +274,6 @@ def test_throughput_tracking(logger):
     check("count_throughput" in all_throughputs["nested"], True)
 
 
-def test_has_throughput_property(logger):
```

Comment:
This slipped through the cracks. Throughputs are now tested in another test case and we don't need the "has throughput" property anymore.